dali/test/python/test_pipeline.py

```diff
@@ -574,18 +574,17 @@ def define_graph(self):
             )
             return (output, self.labels)
 
-    n = 30
     orig_pipe = HybridPipe(batch_size=batch_size, num_threads=2, device_id=0)
     s = orig_pipe.serialize()
-    for i in range(50):
+    for i in range(10):
         pipe = Pipeline()
         pipe.deserialize_and_build(s)
         pipe_out = pipe.run()
         pipe_out_cpu = pipe_out[0].as_cpu()
-        img_chw_test = pipe_out_cpu.at(n)
         if i == 0:
-            img_chw = img_chw_test
-        assert np.sum(np.abs(img_chw - img_chw_test)) == 0
+            ref = pipe_out_cpu
+        else:
+            check_batch(pipe_out_cpu, ref)
```

Comment:
This test checked too little and took too long - now it's more accurate and much faster.